apis/github/code/index.js

```diff
@@ -0,0 +1,74 @@
+
+if (process.argv.length !== 3) {
+    console.error('Usage: node index.js <command>');
+    process.exit(1);
+}
+
+const command = process.argv[2];
+const token = process.env.GITHUB_TOKEN;
+
+if (!token) {
+    console.error('GITHUB_TOKEN environment variable must be set.');
+    process.exit(1);
+}
+
+const octokit = new Octokit({ auth: token });
+
+switch (command) {
+    case 'listIssues':
+        await listIssues(octokit, process.env.OWNER, process.env.REPO);
+        break;
+    case 'listPRs':
+        await listPRs(octokit, process.env.OWNER, process.env.REPO);
+        break;
+    case 'createIssue':
+        await createIssue(octokit, process.env.OWNER, process.env.REPO, process.env.TITLE, process.env.BODY);
+        break;
+    case 'modifyIssue':
+        await modifyIssue(octokit, process.env.OWNER, process.env.REPO, process.env.ISSUENUMBER, process.env.NEWTITLE, process.env.NEWBODY);
+        break;
+    case 'deleteIssue':
+        await deleteIssue(octokit, process.env.OWNER, process.env.REPO, process.env.ISSUENUMBER);
+        break;
+    case 'searchIssues':
+        await searchIssues(octokit, process.env.OWNER, process.env.REPO, process.env.QUERY);
+        break;
+    case 'createPR':
+        await createPR(octokit, process.env.OWNER, process.env.REPO, process.env.TITLE, process.env.BODY, process.env.HEAD, process.env.BASE);
+        break;
+    case 'modifyPR':
+        await modifyPR(octokit, process.env.OWNER, process.env.REPO, process.env.PRNUMBER, process.env.NEWTITLE, process.env.NEWBODY);
+        break;
+    case 'deletePR':
+        await deletePR(octokit, process.env.OWNER, process.env.REPO, process.env.PRNUMBER);
+        break;
+    case 'addCommentToIssue':
+        await addCommentToIssue(octokit, process.env.OWNER, process.env.REPO, process.env.ISSUENUMBER, process.env.COMMENT);
+        break;
+    case 'addCommentToPR':
+        await addCommentToPR(octokit, process.env.OWNER, process.env.REPO, process.env.PRNUMBER, process.env.COMMENT);
+        break;
+    case 'getIssue':
+        await getIssue(octokit, process.env.OWNER, process.env.REPO, process.env.ISSUENUMBER);
+        break;
+    case 'getPR':
+        await getPR(octokit, process.env.OWNER, process.env.REPO, process.env.PRNUMBER);
+        break;
+    case 'getIssueComments':
+        await getIssueComments(octokit, process.env.OWNER, process.env.REPO, process.env.ISSUENUMBER);
+        break;
+    case 'getPRComments':
+        await getPRComments(octokit, process.env.OWNER, process.env.REPO, process.env.PRNUMBER);
+        break;
+    case 'listRepos':
+        await listRepos(octokit, process.env.OWNER);
+        break;
+    case 'closeIssue':
+        await closeIssue(octokit, process.env.OWNER, process.env.REPO, process.env.ISSUENUMBER);
+        break;
+    default:
+        console.error(`Unknown command: ${command}`);
```

Comment:
I think we want all the `console.error` calls to be `console.log` so they go to stdout and are sent to the LLM.

Is that correct @g-linville?

Reply:
Yes that's correct